docs/migration/enact/migrating-to-enact-4.5.md

```diff
@@ -0,0 +1,209 @@
+
+[Here](https://github.com/enactjs/samples/tree/master/sandstone/pattern-react18-new) is an example app for demonstration.
+Check out the `useTransition` tab.
+
+
+#### Suspense
+`Suspense` lets you declaratively specify the loading state for a part of the component tree if it’s not yet ready to be displayed:
+```js
+function ProfilePage() {
+    return (
+        <Suspense fallback={<Spinner />}>
+            <ProfileDetails />
+            <Suspense fallback={<Spinner />}>
+                <ProfileTimeline />
+            </Suspense>
+        </Suspense>
+    );
+}
+```
+Let's look at the example app from [here](https://github.com/enactjs/samples/tree/master/sandstone/pattern-react18-new) at the `Suspense` tab.  
+We have two panels, one with `Suspense`, one without. They both load the same list of images. On the first panel, where we have implemented `Suspense`, we can see that until the data is available, we display a skeleton page that has the exact visual structure of the page with placeholders for the lazy loading data. This offers a more pleasant UI experience. As opposed to it, on the second panel, where we haven't implemented `Suspense`, we can observe that it takes several seconds for content to show on the page. During this time user sees a blank page that might be confusing.
+
+So far, we took around for key Concurrent Features of React 18, other than this, React 18 introduces new hooks like `useId`, `useDeferredValue`, etc.  
+If you want more information, please refer to [How to Upgrade to React 18](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html) and other great articles from the official [React Blog](https://reactjs.org/blog).
+
+
+### cli
+`@enact/cli` must be upgraded to version `5.0.0-alpha.1` or newer.  
+`@enact/cli` `5.0.0-alpha.1` updates the `webpack` to `5.x`, `eslint` to `8.x`, `jest` to `27.x`,
+`react`, `react-dom` to `18.x`, and drops the support of `enzyme`.  
+Developers should ensure their code does not rely on features that are no longer available in these versions.
+
+Please do not hesitate to replace `enzyme` with [@testing-library/react](https://testing-library.com/docs/react-testing-library/intro/) when you make your unit tests.
+
+As `@enact/cli` updates `react` and `react-dom` to `18.x`, `PrerenderPlugin` will automatically
+convert `ReactDOMClient.createRoot` to `ReactDOMClient.hydrateRoot` for prerendered apps.  
+Please make sure to follow the above new `createRoot` API pattern to work prerendering properly.
+
+As we update to `eslint 8`, some of the lint rules could be changed. If you run into unknown lint warnings or errors, don't be afraid, and please proceed to fix them. They are likely to be the rules from [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react), so refer to the console message and look up which rule is related.  
+The in-editor-linting will work just fine if you followed the guide from migration to 4.0.  
```

Comment:
What do you mean exactly? `if you followed the guide from migration to 4.0`?

Reply:
I meant you will be fine if you've done works to make in-editor-linting before.
But I just realized we haven't mentioned this in the migration guides before.
I must have been confused with the internal developer guide. Maybe we need to remove the in-editor-linting part or write it more in detail.